server/routes/notification.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createNotification,
+  markNotificationRead,
+  getAllNotifications,
+} = require("../controllers/notification");
+
+router.route("/create").post(protect, createNotification);
+
+router.route("/mark-read").post(protect, markNotificationRead);
```

Comment:
We should use a [route param](https://expressjs.com/en/guide/routing.html#route-parameters) here to identify which notification to update. You could also consider using `PATCH` instead of `POST`, but there isn't a strong consensus there ([further reading](https://restfulapi.net/rest-put-vs-post/)).